static/app/components/charts/intervalSelector.spec.tsx

```diff
@@ -29,9 +29,14 @@ describe('IntervalSelector', function () {
       />
     );
     render(intervalSelector);
+
+    await waitFor(() => {
+      expect(screen.getByText('Interval')).toBeInTheDocument();
+    });
```

Comment:
we have to wait until rendering is “finished”, otherwise, we get `act` warnings / errors now.